ruby/person.rb

```diff
@@ -0,0 +1,20 @@
+class Person
+  attr_accessor :name, :age
+  attr_reader :id
+
+  def initialize(age, _parent_permission, name = 'Unknownn')
+    @id = rand(1..100)
+    @name = name
+    @age = age
+  end
+
+  def can_use_services?
+    true if age >= 18
+  end
+
+  private
+
+  def of_age?
+    false if age >= 18
```

Comment:
- [ ] This method should return true if age >= 18 not false, please modify it. Also, you can reuse it in the `can_use_services?` method to prevent duplicating code 😄